manifests/m/mspearpoint/Evacuationz/2.13.1/mspearpoint.Evacuationz.locale.en-US.yaml

```
@@ -0,0 +1,9 @@
+PackageIdentifier: mspearpoint.Evacuationz
+PackageVersion: 2.13.1
+PackageLocale: en-US
+Publisher: mspearpoint
```

```suggestion
Publisher: Michael Spearpoint
```
This must match Control Panel.